fast-af/fast-checkout-woocommerce
# Reorganize and rename files

# Description

Reorganize and rename files to create better separation of file types. This new file naming convention and file structure will provide more flexibility as we add new features and update existing features in the future.

# Testing instructions

This new structure has been uploaded to https://fasttestdev.wpcomstaging.com/. Visit this site and verify that all existing features are still working as expected. This update should not have changed any functionality in the plugin.

# Checks
